README.md

```diff
@@ -14,6 +14,32 @@ kubectl config use-context docker-desktop
 helm install dev helm -f helm/values.yaml -f helm/values.local.yaml
 ```
 
+## Accessing Logs with the Sidecar Container
+
+This guide will help you access the log files from the `notifybc` application using a sidecar container. The sidecar is intended to be used only when you need to inspect logs and is not automatically deployed with the application.
+
+1. Deploy the Sidecar:
```

Comment:
Maybe add that these should be run from the `helm/` directory.

Reply:
Nice catch, thanks for taking the time to run through the instructions!